lib/seahorn/SolverBmc.cc

```diff
@@ -0,0 +1,449 @@
+#include "seahorn/SolverBmc.hh"
+#include "seahorn/Transforms/Instrumentation/ShadowMemDsa.hh"
+#include "seahorn/UfoOpSem.hh"
+
+#include "llvm/IR/DebugInfo.h"
+#include "llvm/IR/DebugLoc.h"
+#include "llvm/Support/CommandLine.h"
+
+#include "boost/container/flat_set.hpp"
+#include "seahorn/Expr/ExprLlvm.hh"
+#include "seahorn/Expr/HexDump.hh"
+#include "seahorn/Support/SeaDebug.h"
+
+#ifdef WITH_YICES2
+#include "seahorn/Expr/Smt/Yices2SolverImpl.hh"
+#endif
+
+static llvm::cl::opt<bool>
+    SDumpHex("horn-bmc-solver-hexdump",
+             llvm::cl::desc("Dump memory state using hexdump"),
+             cl::init(false));
+static llvm::cl::opt<std::string>
+    SBmcSmtTactic("horn-bmc-solver-tactic",
+                  llvm::cl::desc("Z3 tactic to use for BMC"),
+                  cl::init("default"));
+static llvm::cl::opt<std::string>
```

Comment:
should be adjusted for yices. Logic we use now is QF_AUFBV

Reply:
if you don't declare option static, you can use it in a different file by declaring it using `extern`.
We have examples of that. So just use the same flag as in `Bmc.cc` to get the logic